db/migrate/20190717030304_add_password_digest_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPasswordDigestToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :password_digest, :string
```

Comment:
Same with the `email` attribute, this `password_digest` can also be added to the
**users table creation** (`db/migrate/20190717025920_create_users.rb`)